docs/contents/components/data-display/data-list/index.mdx

```diff
@@ -1,7 +1,469 @@
 ---
-label: Planned
 title: DataList
-description: We are currently considering the development of `DataList`.
+description: "`DataList` is used to display a list of data items."
```

Comment:
@hirotomoyamada 
Would 'similar' be necessary before 'data items'?

Reply:
@taiseiiiii 

The text is similar for other UI libraries!

https://www.chakra-ui.com/docs/components/data-list